processes/local_development/seed_test_data.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, *args):
             'uuid' : uuid4(),
             'date_created' : datetime.now(timezone.utc).replace(tzinfo=None),
             'date_modified' :datetime.now(timezone.utc).replace(tzinfo=None),
-            'frbr_status': 'to_do',
+            'frbr_status': 'complete',
             'cluster_status': False,
             "source": 'test data source',
```

Comment:
Recommend pulling this up as a constant outside the class. Let's use something that we can use in other fields as well, e.g. 

TEST_SOURCE = 'test_source'